akka-http-core/src/test/scala/akka/http/engine/client/OutgoingConnectionSpec.scala

```diff
@@ -359,13 +359,13 @@ class HttpClientSpec extends AkkaSpec("akka.loggers = []\n akka.loglevel = OFF")
       val netOut = StreamTestKit.SubscriberProbe[ByteString]
       val netIn = StreamTestKit.PublisherProbe[ByteString]
 
-      FlowGraph.closed(HttpClient.clientBlueprint(remoteAddress, settings, NoLogging)) { implicit b ⇒
+      FlowGraph.closed(OutgoingConnectionBlueprint(remoteAddress, settings, NoLogging)) { implicit b ⇒
         client ⇒
           import FlowGraph.Implicits._
-          Source(netIn) ~> client.bytesIn
-          client.bytesOut ~> Sink(netOut)
-          Source(requests) ~> client.httpRequests
-          client.httpResponses ~> Sink(responses)
+          Source(netIn) ~> client.in2
```

Comment:
hm, maybe we can use a nicer API here. Maybe something like (Sink, Source) <~> client <~> (Sink, Source), or alternatively Flow(sink, source) <~> client <~> Flow(sink, source)